### PR TITLE
doc: Add doc for fault-injection plugin.

### LINKED ISSUE #1052: add doc for fault injection plugin
> missing doc about how to use this plugin. we can submit it in a separate new PR.

OK. I will submit the documentation later.

_Originally posted by @agile6v in https://github.com/apache/incubator-apisix/pull/1042#issuecomment-573669558_

### PR DESCRIPTION
**Pull request by agile6v**
### Issues resolved
Fix #1052

### CODE REVIEW

File: doc/plugins/fault-injection-cn.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+
+故障注入插件，该插件可以和其他插件一起使用，并且会在其他插件前被执行，配置 `abort` 参数将直接返回给客户端指定的响应码并且终止其他插件的执行，配置 `delay` 参数将延迟某个请求，并且还会执行配置的其他插件。
+
+### 参数
+
+|名称    |必须|描述|
+|------- |-----|------|
+|abort|否|参数 http_status ：必须，返回给客户端的 http 状态码|
+|||参数 body ：可选，返回给客户端的响应数据|
+|delay|否|参数 duration ：必须，延迟时间，可以指定小数|
```
Comment (membphis) [inline doc/plugins/fault-injection-cn.md]: How about this code style?

|名称    |必须|描述|
|------- |-----|------|
|abort.http_status|可选|返回给客户端的 http 状态码，默认 200|
|abort.body|可选|返回给客户端的响应数据|
|delay.duration|可选|延迟时间，可以指定小数|

File: doc/plugins/fault-injection-cn.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+curl http://127.0.0.1:9080/apisix/admin/routes/1 -X PUT -d '
+{
+    "plugins": {
+	   "fault-injection": {
+	       "abort": {
+	          "http_status": 200,
+	          "body": "Fault Injection!"
+	       }
+	   },
+	   "proxy-rewrite": {
```
Comment (membphis) [inline doc/plugins/fault-injection-cn.md]: it should be useless, we can remove it.

File: doc/plugins/fault-injection-cn.md [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+
+#### 启用插件
+
+示例1：为特定路由启用 `fault-injection` 插件，并指定 `abort` 参数：
+
+```shell
+curl http://127.0.0.1:9080/apisix/admin/routes/1 -X PUT -d '
+{
+    "plugins": {
+	   "fault-injection": {
```
Comment (membphis) [inline doc/plugins/fault-injection-cn.md]: Replace `tab` with spaces.

**Comment by membphis** [comment]
Please resolve the English documentation for the same issue

File: doc/plugins/fault-injection-cn.md [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+curl http://127.0.0.1:9080/apisix/admin/routes/1 -X PUT -d '
+{
+    "plugins": {
+       "fault-injection": {
+           "delay": {
+              "duration": 3
+           }
+       }
+    },
+    "upstream": {
```
Comment (membphis) [inline doc/plugins/fault-injection-cn.md]: Can we delete this field too? It seems useless too.

File: doc/plugins/fault-injection-cn.md [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+#### 禁用插件
+
+移除插件配置中相应的 JSON 配置可立即禁用该插件，无需重启服务：
+
+```shell
+curl http://127.0.0.1:9080/apisix/admin/routes/1 -X PUT -d '
+{
+    "uri": "/hello",
+    "plugins": {},
+    "upstream": {
```
Comment (membphis) [inline doc/plugins/fault-injection-cn.md]: ditto

File: doc/plugins/fault-injection-cn.md [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+
+# fault-injection
+
+故障注入插件，该插件可以和其他插件一起使用，并且会在其他插件前被执行，配置 `abort` 参数将直接返回给客户端指定的响应码并且终止其他插件的执行，配置 `delay` 参数将延迟某个请求，并且还会执行配置的其他插件。
+
+### 参数
+
+|名称    |必须|描述|
+|------- |-----|------|
+|abort.http_status|是|返回给客户端的 http 状态码|
```
Comment (membphis) [inline doc/plugins/fault-injection-cn.md]: I think it can be set a default value 200 if the user not specified.

Comment (moonming) [reply doc/plugins/fault-injection-cn.md]: I don't think so. This plugin is for `fault` injection, `200` is not fault return.

File: doc/plugins/fault-injection-cn.md (hunk 4 above)
Comment (agile6v) [reply doc/plugins/fault-injection-cn.md]: It's necessary for the delay attribute.

File: doc/plugins/fault-injection-cn.md (hunk 6 above)
Comment (agile6v) [reply doc/plugins/fault-injection-cn.md]: If the user specify neither http_status nor body, seems that the readability is not good.

**Pull request merged**